### PR TITLE
fix links and other updates in cloud section

### PR DESCRIPTION
**Pull request by dylangottlieb**

### CODE REVIEW

**Review by b-reyes — CHANGES_REQUESTED** [review]
Your changes look good. I will ask one annoying thing though. Could you please replace all the statements like this:
```
<a href="https://docs.aws.amazon.com/cost-management/latest/userguide/create-cost-budget.html" target="_blank">AWS Cost Management console</a>
```
with markdown syntax? This will make our documentation consistent and will help us catch any broken links for internally referenced docs.

File: docs/cloud/aws/budget-actions/budget-actions.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ You can attach budget actions to an alert for either a cost budget or a usage bu
 5.	Choose ‘Budget scope’ and select the services you would like in scope. For this example we will choose 'All AWS Services' and choose ‘Next’
     ![](https://raw.githubusercontent.com/ResearchComputing/Documentation/master/docs/cloud/aws/budget-actions/images/3.png)
```
Comment (b-reyes) [inline docs/cloud/aws/budget-actions/budget-actions.md:20]: This is a strange thing. Can we please remove this link and use a proper Markdown reference. Could you please do this for any other items that are like this?

File: docs/cloud/aws/getting-started/aws-console-access.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # Accessing the AWS Management Console
 
 ## Prerequisites
-1. You are a member of CUBoulder. If you have access to CUBoulder AWS resources and are from another institution, please see <a href="./aws-console-access-non-cu-boulder.md" target="_blank">this guide</a> on signing in using your institution's credentials.
+1. You are a member of CUBoulder. If you have access to CUBoulder AWS resources and are from another institution, please see [this guide](non-cu-boulder/aws-console-access-non-cu-boulder.html) on signing in using your institution's credentials.
```
Comment (b-reyes) [inline docs/cloud/aws/getting-started/aws-console-access.md:4]: For internal references, you should not use `.html`, instead, `.md` should be used.

File: docs/cloud/aws/getting-started/non-cu-boulder/aws-cli-saml2aws-non-cu-boulder.md [hunk 3]
```diff
@@ -4,13 +4,13 @@ The **saml2aws** CLI tool is used to login and retrieve AWS temporary credential
 
 ## Prerequisites
 
-1. This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see <a href="./aws-console-access.md" target="_blank">this guide</a> on signing in using your IdentiKey.
+1. This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see [this guide](../aws-cli-saml2aws.html) on signing in using your IdentiKey.
```
Comment (b-reyes) [inline docs/cloud/aws/getting-started/non-cu-boulder/aws-cli-saml2aws-non-cu-boulder.md:7]: For internal references, you should not use `.html`, instead, `.md` should be used.

File: docs/cloud/aws/getting-started/non-cu-boulder/aws-console-access-non-cu-boulder.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 # Accessing the AWS Management Console with CILogon (Non-CU Boulder)
 
 ## Prerequisites
-This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see <a href="./aws-console-access.md" target="_blank">this guide</a> on signing in using your IdentiKey.
+This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see [this guide](../aws-console-access.html) on signing in using your IdentiKey.
```
Comment (b-reyes) [inline docs/cloud/aws/getting-started/non-cu-boulder/aws-console-access-non-cu-boulder.md:4]: For internal references, you should not use `.html`, instead, `.md` should be used.

**Review by b-reyes — CHANGES_REQUESTED** [review]

File: docs/cloud/aws/getting-started/non-cu-boulder/aws-console-access-non-cu-boulder.md (hunk 4 above)
Comment (dylangottlieb) [reply docs/cloud/aws/getting-started/non-cu-boulder/aws-console-access-non-cu-boulder.md]: ccb0f5c

File: docs/cloud/aws/getting-started/non-cu-boulder/aws-cli-saml2aws-non-cu-boulder.md (hunk 3 above)
Comment (dylangottlieb) [reply docs/cloud/aws/getting-started/non-cu-boulder/aws-cli-saml2aws-non-cu-boulder.md]: ccb0f5c

File: docs/cloud/aws/getting-started/aws-console-access.md (hunk 2 above)
Comment (dylangottlieb) [reply docs/cloud/aws/getting-started/aws-console-access.md]: ccb0f5c

File: docs/cloud/aws/budget-actions/budget-actions.md (hunk 1 above)
Comment (dylangottlieb) [reply docs/cloud/aws/budget-actions/budget-actions.md:20]: Isn't this the correct markdown for images?

Comment (b-reyes) [reply docs/cloud/aws/budget-actions/budget-actions.md:20]: Markdown syntax is fine. What I am referencing is the use of the GitHub link. This is an image that is in our docs, so it should be internally referenced.

Comment (dylangottlieb) [reply docs/cloud/aws/budget-actions/budget-actions.md]: 436c85e

**Comment by dylangottlieb** [comment]
> Your changes look good. I will ask one annoying thing though. Could you please replace all the statements like this:
> 
> ```
> <a href="https://docs.aws.amazon.com/cost-management/latest/userguide/create-cost-budget.html" target="_blank">AWS Cost Management console</a>
> ```
> 
> with markdown syntax? This will make our documentation consistent and will help us catch any broken links for internally referenced docs.

[f62a111]

File: docs/cloud/aws/budget-actions/budget-actions.md [hunk 5]
```diff
@@ -55,16 +55,16 @@ This managed policy is focused on specific actions that AWS Budgets takes on you
...
 12. For **How do you want to be alerted when this action is run, choose Use the same alert settings when you defined this threshold** or **Use different alert settings**. To use different alert settings, complete the **Notification preferences** specific to this action.
-    ![](https://raw.githubusercontent.com/ResearchComputing/Documentation/master/docs/cloud/aws/budget-actions/images/7.png)
+    ![](images/7.png)
 
 
 13.	Choose **Next**.
 
 14.	Review your budget settings, and then choose **Create budget**.
-    ![](https://raw.githubusercontent.com/ResearchComputing/Documentation/master/docs/cloud/aws/budget-actions/images/8.png)
+    ![](images/8.png)
```
Comment (b-reyes) [inline docs/cloud/aws/budget-actions/budget-actions.md:64]: Something weird is happening here in the rendering. For all the images you work with, I would make sure the rendering looks good. 

Screenshot 2024-09-06 at 4 19 52 PM

**Review by b-reyes — CHANGES_REQUESTED** [review]

File: docs/cloud/aws/budget-actions/budget-actions.md (hunk 5 above)
Comment (dylangottlieb) [reply docs/cloud/aws/budget-actions/budget-actions.md:64]: Screenshot 2024-09-06 at 4 23 30 PM

Comment (dylangottlieb) [reply docs/cloud/aws/budget-actions/budget-actions.md]: b9fea6a

Comment (dylangottlieb) [reply docs/cloud/aws/budget-actions/budget-actions.md]: Fixed by adding some new lines

Comment (dylangottlieb) [reply docs/cloud/aws/budget-actions/budget-actions.md]: 11d9cc0

Comment (dylangottlieb) [reply docs/cloud/aws/budget-actions/budget-actions.md]: Final fix ^^

**Review by b-reyes — CHANGES_REQUESTED** [review]
Looked through all of the pages a couple more items to note.
-  It looks like the page `cloud/azure/budget-actions/budget-actions.md` also has rendering issues on larger screens
- Someone put a custom highlighting that is not viewable in the dark mode:
Screenshot 2024-09-06 at 4 46 11 PM

**Comment by b-reyes** [comment]
For the custom highlighting issue, I suggest replacing it with natively supported boxes: 

````
```{important}

Blah blah text

```

````

**Comment by b-reyes** [comment]
For example. replace: 

```
<table style="background-color: #dcdcdc; filter: alpha(opacity=40); opacity: 0.95;">
<tr>
<td>
<b>( ! ) - IMPORTANT</b>
</td>
</tr>
<tr>
<td>
Note that when using some AWS wizards to create services, the wizard will try to create a Role.
Due to the limitations discussed below, the wizard will fail.
When encountering this situation, you must manually create the Role with the proper permissions, and attach the permission boundary discussed below.
Then, in the wizard, you need to pick the Role you created.
</td>
</tr>
</table>
<br>
```

with 
````
```{important}
Note that when using some AWS wizards to create services, the wizard will try to create a Role.
Due to the limitations discussed below, the wizard will fail.
When encountering this situation, you must manually create the Role with the proper permissions, and attach the permission boundary discussed below.
Then, in the wizard, you need to pick the Role you created.
```
````

**Review by b-reyes — APPROVED** [review]

**Pull request merged**